perf/benchmarkapps/Shared/GrpcCoreGreet.proto

```diff
@@ -0,0 +1,37 @@
+// Copyright 2019 The gRPC Authors
```

Comment:
Can't have the same file with different arguments?

nit: Rename to greet-core.proto to it is more consistent with other proto files.

Reply:
I think there would be typename conflicts since they are in the same namespace, hence why I changed the filename. I can control the output directory via msbuild metadata but I can't control the typename and namespace without actually changing the proto file.